app/welcome/route.js

```diff
@@ -1,5 +1,11 @@
 import Ember from 'ember';
 
+export function resetDBData(model){
+  Ember.set(model, 'dbType', null);
+  Ember.set(model, 'initialDiskSize', 10);
```

Comment:
@mixonic What is the advantage of using `Ember.set` versus just `model.set` ?  I'm also curious if `resetDBData` would be better as an instance method on the database model?